frontend/cypress/integration/featureFiles/graph_display.feature

```diff
@@ -181,3 +181,10 @@ Scenario: User enables animation
 Scenario: User disables animation
   When user "disables" "traffic animation" option
   Then "traffic animation" option "does not appear" in the graph
+
+@graph-page-display
+Scenario: User resets to factory default
+  When user resets to factory default
+  And user opens display menu
+  Then the display menu opens
```

Comment:
This is a bit redundant. Checking that the display menu actually opened can go in the implementation for `And user opens display menu`.